docs/ref/settings.txt

```diff
@@ -2644,6 +2644,11 @@ A ready-to-use instance of the storage backends can be retrieved from
 :data:`django.core.files.storage.storages`. Use a key corresponding to the
 backend definition in :setting:`STORAGES`.
 
+.. admonition:: How is my value merged with the default value?
```

Comment:
Personally, I don't like this title because it suggests that the value is merged.

Reply:
How about this?

```suggestion
.. admonition:: Is my value merged with the default value?
```

Reply:
> How about this?

Better :+1: thanks.